Doc/using/windows.rst

```diff
@@ -602,6 +602,50 @@ existed)::
 
     C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Python 3.9
 
+.. _win-utf8-mode:
+
+UTF-8 mode
+==========
+
+.. versionadded:: 3.7
+
+Windows still uses legacy encodings for the system encoding (the ANSI Code
+Page).  Python uses it for the default encoding of text files (e.g.
+:func:`locale.getpreferredencoding`).
+
+It may cause trouble because the UTF-8 is widely used on the internet
```

Comment:
It would read better as "This may" instead of "It may". Using "this" is more clearly referring to the previous statement.